EnglishFilter/sections/specific.txt

```diff
@@ -265,7 +265,7 @@ adsy.pw##iframe[id^="adsy.pw_"][id*="_DFP"]
 ping.eu##td > center[style="margin-bottom: 2px;"]
 hltv.org##.bitskins
 wheelofnames.com##.ad-declaration
-||hentaila.com/files/test.php
+||hentaila.com/*.php$script,~third-party
```

Comment:
This is a Spanish site. You also block the consequences, but not the reason for the ad.
Please have a look at this `||hentaila.com/files/functions.js`. You should exchange your rule to this one:)

Reply:
Done. I really have to be careful to site's language :(

Reply:
> Done. I really have to be careful to site's language :(

This will come with experience :)